src/utils/event-utils.ts

```diff
@@ -70,261 +70,205 @@ export const EVENT_TO_PERSON_PROPERTIES = [
 
 export const MASKED = '<masked>'
 
-export const Info = {
-    campaignParams: function ({
-        customTrackedParams,
-        maskPersonalDataProperties,
-        customPersonalDataProperties,
-    }: {
-        customTrackedParams?: string[]
-        maskPersonalDataProperties?: boolean
-        customPersonalDataProperties?: string[] | undefined
-    } = {}): Record<string, string> {
-        if (!document) {
-            return {}
-        }
+export function getCampaignParams(
+    customTrackedParams?: string[],
+    maskPersonalDataProperties?: boolean,
+    customPersonalDataProperties?: string[] | undefined
+): Record<string, string> {
+    if (!document) {
+        return {}
+    }
 
-        const paramsToMask = maskPersonalDataProperties
-            ? extendArray([], PERSONAL_DATA_CAMPAIGN_PARAMS, customPersonalDataProperties || [])
-            : []
+    const paramsToMask = maskPersonalDataProperties
+        ? extendArray([], PERSONAL_DATA_CAMPAIGN_PARAMS, customPersonalDataProperties || [])
+        : []
 
-        return this._campaignParamsFromUrl(maskQueryParams(document.URL, paramsToMask, MASKED), customTrackedParams)
-    },
+    return _getCampaignParamsFromUrl(maskQueryParams(document.URL, paramsToMask, MASKED), customTrackedParams)
+}
 
-    _campaignParamsFromUrl: function (url: string, customParams?: string[]): Record<string, string> {
-        const campaign_keywords = CAMPAIGN_PARAMS.concat(customParams || [])
+function _getCampaignParamsFromUrl(url: string, customParams?: string[]): Record<string, string> {
```

Comment:
Does it even make sense to prefix them with an underscore? Not exporting them should be enough to indicate they're private

Reply:
Yeah agreed, I'll do that in the follow-up